api/comment_threads.rb

```diff
@@ -1,21 +1,22 @@
 
 get "#{APIPREFIX}/threads" do # retrieve threads by course
+
   threads = CommentThread.where({"course_id" => params["course_id"]})
```

Comment:
The quotes changes are mostly personal preference: single quotes for everything except interpolation or strings containing single quotes. A similar best practice could apply to Python; but, I don't think we've established such a practice outside of individual teams.